Doppler.UsersApi/Infrastructure/AccountRepository.cs

```diff
@@ -0,0 +1,44 @@
+using Dapper;
+using Doppler.UsersApi.Model;
+using System.Data;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Doppler.UsersApi.Infrastructure
+{
+    public class AccountRepository : IAccountRepository
+    {
+        private readonly IDatabaseConnectionFactory _connectionFactory;
+        public AccountRepository(IDatabaseConnectionFactory connectionFactory)
+        {
+            _connectionFactory = connectionFactory;
+        }
+        public async Task<ContactInformation> GetContactInformation(string email)
+        {
+            using (IDbConnection connection = await _connectionFactory.GetConnection())
+            {
+                var results = await connection.QueryAsync<ContactInformation>(@"
+                    SELECT
+                        U.FirstName,
+                        U.LastName,
+                        U.Email,
+                        U.IdIndustry AS Industry,
+                        U.Company, U.PhoneNumber AS Phone,
+                        U.Address, U.ZipCode,
+                        CI.Name AS City,
+                        U.CityName,
+                        S.Name AS Province,
+                        CO.Code AS Country
+                    FROM
+                        [User] U
+                        INNER JOIN [State] S ON U.IdState = S.IdState AND U.IdBillingState = S.IdState
+                        INNER JOIN City CI ON S.IdState = CI.IdState
+                        INNER JOIN [Country] CO ON S.IdCountry = CO.IdCountry
+                    WHERE
+                        U.Email = @email",
```

Comment:
It is to avoid a lot of unnecessary spaces that will give us issues if we use a profiler:

```suggestion
                var results = await connection.QueryAsync<ContactInformation>(@"
SELECT
    U.FirstName,
    U.LastName,
    U.Email,
    U.IdIndustry AS Industry,
    U.Company, U.PhoneNumber AS Phone,
    U.Address, U.ZipCode,
    CI.Name AS City,
    U.CityName,
    S.Name AS Province,
    CO.Code AS Country
FROM
    [User] U
    INNER JOIN [State] S ON U.IdState = S.IdState AND U.IdBillingState = S.IdState
    INNER JOIN City CI ON S.IdState = CI.IdState
    INNER JOIN [Country] CO ON S.IdCountry = CO.IdCountry
WHERE
    U.Email = @email",
```